ui/plugins/eu.esdihumboldt.hale.ui/src/eu/esdihumboldt/hale/ui/io/instance/crs/SelectCRSDialog.java

```diff
@@ -309,13 +309,29 @@ public void widgetSelected(SelectionEvent e) {
 		if (value instanceof CodeDefinition) {
 			code = ((CodeDefinition) value).getCode();
 		}
+		else if (value instanceof WKTDefinition) {
+			// Don't show possibly contradicting code
+			code = "";
+		}
 		else {
 			code = lastCode;
 		}
 		crsField.setStringValue(code);
 		crsField.setPropertyChangeListener(this);
 		crsField.setEnabled(lastWasCode, group);
 
+		new Label(group, SWT.NONE);
+
+		wktWarning = new Label(group, SWT.NONE);
+		wktWarning.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, false, 2, 1));
+		wktWarning.setText(
+				"Please be aware that if you provide a WKT definition here, hale studio may\n"
+						+ "not be able to perform accurate coordinate transformations due to missing\n"
+						+ "Bursa-Wolf parameters. If you intend to perform coordinate transformations,\n"
```

Comment:
Yes, that's possible ([`TOWGS84`](http://docs.geotools.org/stable/javadocs/org/opengis/referencing/doc-files/WKT.html#TOWGS84)). I can add a check so that the warning is only shown if they are actually not provided.

Reply:
I modified the message a bit and it's now displayed only if there are no _BWP_ in the WKT definition.